docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,348 @@
+
+1. Have a postgres 9 or postgres 10 database cluster
+2. The database cluster must have the following databases
+   1. defaultdb
+   2. accounts
+   3. analysis
+   4. results
+   5. metrics
+   6. filestore
+   7. jobs
+3. In the values file that you provide during the `helm` installation, define the following for the databases described above:
+   1. `global.<database>db.create=false` - disable the creation of this database inside the cluster.
+   2. `global.<database>db.postgresqlUsername` - the username that will have access to this database.
+   3. `global.<database>db.postgresqlPassword` - the passworf for the given user.
+   4. `global.<database>db.host` - hostname of the external database.
+4. The database cluster must have the following databases
+   1. activities
+   2. hotspots
+   3. listener
+5. In the values file that you provide during the `helm` installation, define the following for the databases described above:
+   1. `<database>db.create=false` - disable the creation of this database inside the cluster.
+   2. `<database>db.postgresqlUsername` - the username that will have access to this database.
+   3. `<database>db.postgresqlPassword` - the passworf for the given user.
+   4. `<database>db.host` - hostname of the external database.
+6. Note that the provided user must be the `owner` of the database(s):
+    >ALTER ROLE \<user> WITH CREATEDB;
+
+    Then you can connect as that user and create the databases:
+    ```sql
+    CREATE DATABASE accounts WITH OWNER=<user>;
+    CREATE DATABASE analysis WITH OWNER=<user>;
+    CREATE DATABASE results WITH OWNER=<user>;
+    CREATE DATABASE metrics WITH OWNER=<user>;
+    CREATE DATABASE filestore WITH OWNER=<user>;
+    CREATE DATABASE jobs WITH OWNER=<user>;
+    CREATE DATABASE activities WITH OWNER=<user>;
+    CREATE DATABASE hotspots WITH OWNER=<user>;
+    CREATE DATABASE listener WITH OWNER=<user>;
+    ```
+
+## 6. Accessing the Codacy UI
+
+In the previous section, you should have defined values for `codacy-api.ingress.hosts.host`, `global.codacy.url`, and `global.codacy.backendUrl`. This value will be the address where you can access the Codacy UI from your web browser.
+
+## 7. Troubleshooting
+
+### 1. AWS EC2 specific issues
+
+1. We had to configure the instance's list of hosts with localhost. This was because our machine was in fact an AWS EC2 instance, and therefore we must allow the correct resolution of the local microk8s cluster on the kubeconfig. We did so by running
+
+    >sudo sed -i -e "1s/127.0.0.1 localhost.*/127.0.0.1 localhost $(hostname)/" /etc/hosts
+
+2. Traffic forwarding must be configured on the machine so that the local microk8s cluster pods can reach the internet, again because of using an AWS EC2 instance. This can be done by running
+    >sudo iptables -P FORWARD ACCEPT
+
+    (more info on this subject can be [found here](https://microk8s.io/docs/troubleshooting#common-issues)).
+
+## microk8s-values.yaml
```

Comment:
See comment above about providing this as a standalone file. :)

Reply:
I am not 100% sure what to do about it :/ suggestions?

Reply:
I confused the files in my comment above, sorry.

What I meant is that we are providing the following file in the GitHub repository itself:

https://github.com/codacy/chart/blob/master/codacy/values-production.yaml

We could also provide this `microk8s-values.yaml` in the same way so that it's a bit easier for users to download and use it - and it also makes this documentation shorter.

I suggest calling the file `values-production-microk8s.yaml` so it is ordered alphabetically and stays close to the `values-production.yaml` one.

Reply:
Good idea. I wouldn't call it 'production' however. It is mostly just a placeholder/template/starting point. But i have moved it to live alongside the other values files :)